### ISSUE TITLE
Select and copy text

### ISSUE THREAD

**Issue by var314** [dunst-project/dunst#988]
Hi, is it possible to select text in notification by mouse and copy/paste it?
#246 is marked as solved by #530, which would suggest it is possible. But it does not work for me. I tried with `mouse_left_click = none` and with its default `close_current`. I also tried `force_xwayland = true`.

### Installation info

- Version: `1.7.1 (2021-11-01)`
- Install type: `package`
- Distro and version: `Arch Linux`
- Graphical environment: `Sway/Wayland`

**Comment by fwsmit** [comment]
Yeah it would be interesting. I don't know how doable it would be, since we're not using any kind of GUI toolkit. This feature may be more suitable to other notification daemons.

**Comment by joshpetit** [comment]
@fwsmit Are you aware of any other daemon that has this capability?

**Comment by fwsmit** [comment]
No, I'm not aware of any other daemon that can do this. Your workaround seems feasible.

**Comment by bynect** [comment]
Copying the whole summary is easy (xclip in the script) but for specific part of text there is no way to select them

**Comment by eugenevd** [comment]
> Copying the whole summary is easy (xclip in the script) but for specific part of text there is no way to select them

In that case, if the string is of consistent format, you'd probably be able to get the selection with grep or other tools as part of that script

**Comment by dicktyr** [comment]
this is essential

text displayed without the ability to select it is arguably not text

it's a pity that the sad state of information technology in 2025
makes it so onerous that even a simple notification tool
should depend on yet another monstrous GUI toolkit
